testing/constraints-3.7.txt

```diff
@@ -17,6 +17,7 @@ ipython==7.0.1
 opentelemetry-api==1.1.0
 opentelemetry-instrumentation==0.20b0
 opentelemetry-sdk==1.1.0
+packaging==14.3
```

Comment:
@tswast It looks that [The problem you suggest](https://github.com/googleapis/python-bigquery/pull/1440#issuecomment-1358325099) comes from this version constraint.

v14.3 of packaging has no `major` attribute:
https://github.com/pypa/packaging/blob/14.3/packaging/version.py#L268-L280

I think we need to update lower bound, too.

Reply:
`major` is added since v20.0.

ref: https://github.com/pypa/packaging/releases/tag/20.0